### PR TITLE
Add some news of previous and present work

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

File: src/site/xdoc/index.xml [hunk 1]
```diff
@@ -398,6 +398,19 @@
 
       <subsection name="News">
         <ul>
+          <li>Sep/2016 - <ul>
+          <li><a href="newsarchive.html#20160901"><strong>Apache James Server 3.0</strong> We are currently working on the release!!!</a></li>
```
Comment (rouazana) [inline src/site/xdoc/index.xml]: Please, no "!!!" :)

File: src/site/xdoc/newsarchive.xml [hunk 2]
```diff
@@ -36,6 +36,30 @@
 
    <subsection name='2016'>
 
+     <a name="20160901"><h4>Sep 2016 - Apache James Server 3.0</h4></a>
+     <p>We are currently working on the release!!!</p>
```
Comment (rouazana) [inline src/site/xdoc/newsarchive.xml]: idem

File: src/site/xdoc/newsarchive.xml [hunk 3]
```diff
@@ -36,6 +36,30 @@
...
+     <p>Keep an eye on the news, we are planning to release the next version in November.</p>
+
+     <a name="20160702"><h4>July 2016 - JMAP implementation</h4></a>
+     <p>Full text search via JMAP.</p>
+     <p>Full text search via the <i>getMessageList</i> JMAP command is now supported.</p>
+     <p>The JMAP search <b>filtering</b> parameter is converted to an ElasticSearch query; then this query is processed by the ElasticSearch backend.</p>
+
+     <a name="20160701"><h4>July 2016 - JMAP implementation</h4></a>
+     <p>Read and Send message with attachments.</p>
+     <p>You can now send and read messages with attachments thru the JMAP commands <i>setMessages</i> and <i>getMessages</i>.</p>
```
Comment (mbaechler) [inline src/site/xdoc/newsarchive.xml]: through

File: src/site/xdoc/newsarchive.xml [hunk 4]
```diff
@@ -36,6 +36,30 @@
...
    <subsection name='2016'>
 
+     <a name="20160901"><h4>Sep 2016 - Apache James Server 3.0</h4></a>
+     <p>We are currently working on the release!!!</p>
+     <p>Keep an eye on the news, we are planning to release the next version in November.</p>
+
+     <a name="20160702"><h4>July 2016 - JMAP implementation</h4></a>
+     <p>Full text search via JMAP.</p>
+     <p>Full text search via the <i>getMessageList</i> JMAP command is now supported.</p>
+     <p>The JMAP search <b>filtering</b> parameter is converted to an ElasticSearch query; then this query is processed by the ElasticSearch backend.</p>
```
Comment (rouazana) [inline src/site/xdoc/newsarchive.xml]: Technically, that's not an ElasticSearch query, but a whatever backend query, no?

Comment (mbaechler) [inline src/site/xdoc/newsarchive.xml]: nope, an internal SearchQuery

File: src/site/xdoc/newsarchive.xml [hunk 5]
```diff
@@ -36,6 +36,30 @@
 
    <subsection name='2016'>
 
+     <a name="20160901"><h4>Sep 2016 - Apache James Server 3.0</h4></a>
+     <p>We are currently working on the release!!!</p>
+     <p>Keep an eye on the news, we are planning to release the next version in November.</p>
```
Comment (mbaechler) [inline src/site/xdoc/newsarchive.xml]: give a link about that

File: src/site/xdoc/newsarchive.xml (hunk 3 above)
Comment (rouazana) [inline src/site/xdoc/newsarchive.xml]: I think "thru" should not be used in formal presentation. Prefer trough. (same in other places of this PR)

**Comment by aduprat** [comment]
test this please

**Comment by chibenwa** [comment]
After "!!!" fixing you got my :+1:

**Comment by aduprat** [comment]
Done

**Comment by aduprat** [comment]
test this please

**Pull request merged**